### PR TITLE
Beginning

### PR DESCRIPTION
**Pull request by ShayHall**

### CODE REVIEW

**Comment by karjac14** [comment]
Great tutorial in scientific methods, great way to approach big problems. :+1:

**Comment by cmauban** [comment]
Yes, we have way more failures, but ahh when we FINALLY get it to work- best feeling ever. (hopefully we can get there with the chessboard) Nice read. Nice work. You're doing great :+1:  Oh, and good job making the tutorial more clear!

**Comment by boshnivolo** [comment]
Love your journal! I feel like I can definitely relate :+1:

File: scientific_methodizer.md [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
 #### _Replacing the madness of your method with some science._
-<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/Johannes_Kepler_1610.jpg/800px-Johannes_Kepler_1610.jpg" width=25%>
 
-Sticking to the scientific method for the majority of your coding process will save time, energy, confusion, and quite possibly your sanity. Early in our coding lives, everything is new.  This means the world of computers is dark and mysterious, but it also means there are lots of lightbulbs waiting to be turned on.  The **scientific method** will help ensure there are more bright bulbs and goals met rather than just confused shrugs and targets hit and you won't end up like this guy:
+
+Sticking to the scientific method for the majority of your coding process will save time, energy, confusion, and quite possibly your sanity. Early in your coding life, everything is new.  This means the world of computers is dark and scary, but it also means there are lots of lightbulbs waiting to be turned on.  The **scientific method** will help ensure there are more bright bulbs and goals met rather than just confused shrugs and targets hit and you won't end up like this guy:
```
Comment (kellymurray) [inline scientific_methodizer.md]: Interesting tutorial! I'd like to see you tackle a technical objective next time.

**Pull request merged**